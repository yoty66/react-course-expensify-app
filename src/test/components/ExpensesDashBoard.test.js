import React from 'react';
import { shallow } from 'enzyme';
import ExpensesDashBoard  from '../../components/ExpenseDashBoard';

test('should render correctly ' , () =>{
    const wrapper = shallow (<ExpensesDashBoard/>);
    expect(wrapper).toMatchSnapshot();
});