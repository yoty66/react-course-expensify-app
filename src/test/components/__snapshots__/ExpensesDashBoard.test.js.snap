// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly  1`] = `
<div>
  <Connect(ExpensesListSummary) />
  <Connect(ExpenseListFilter) />
  <Connect(ExpenseList) />
</div>
`;
